.github/workflows/check.yml

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Check
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+
+  unit-test:
```

Comment:
can you add a one-line comment above each of these jobs briefly explaining what they do?

Reply:
Sure. I thought it was clear from the name. :)

Reply:
thanks, I'm a lazy reader

Reply:
Hmm. All my descriptions seem complete redundant with the names. Maybe you can tell me what's unclear?

Reply:
I put my suggestions inline in go.yaml :)